cdap-app-fabric/src/main/java/io/cdap/cdap/gateway/handlers/ProgramLifecycleHttpHandler.java

```diff
@@ -1534,9 +1560,7 @@ public void getRunCounts(FullHttpRequest request, HttpResponder responder,
 
     // Get latest version runs in CDAP-19654
```

Comment:
This line can be removed since we're using latest version in this PR

Reply:
removed